services/mozilla-observatory/mozilla-observatory.service.js

```diff
@@ -63,8 +63,7 @@ module.exports = class MozillaObservatory extends BaseJsonService {
     return [
       {
         title: 'Mozilla HTTP Observatory Grade',
-        pattern: 'grade/:host',
-        namedParams: { host: 'github.com' },
+        namedParams: { which: 'grade', host: 'github.com' },
```

Comment:
I'm on the fence about leveraging the dropdowns for this service. My initial thinking when we recently merged this PR was that since the `grade-score` option includes an entirely additional/new data point (the score), that it warrants its own listing on the main page. 

The other services in this PR make sense to me as the dropdown is for a param that's essentially for different filters on the same data set (download period, install targets) or an attribute of the same data entity in the case of the github-issue-detail-service.

Here on the Mozilla Observatory service, it's the same data (the analysis result) but with or without the extra data point (the score)

Reply:
A similar set of these are the rating badges, where there's rating + score. I think it's arguable that it's the same badge with a different format option, and that showing both forms doesn't provide a lot of extra information in the listing that wouldn't be discovered by anyone who needs it, after clicking through. Though I don't feel strongly! I'm happy to remove this one for now.

Reply:
I guess I was only looking at this service in comparison with the others in the PR. Do you know off hand of some of the other rating + score badges? I can think of a few that are one or the other but not combos off the top of my head

Also I don't feel strongly either 😄 I keep going back and forth internally

Reply:
> Do you know off hand of some of the other (configurable) rating + score badges

Mozilla add-on, VS Code Marketplace, Redmine plugin…

None of them show combined info. But there are two different ways of showing the same metric.

Reply:
Yeah this one makes more sense to me if I look at it more as the `short` vs. `long` form of the analysis. 

I've approved as I'm good with dropdowns here.